hw/ip/otbn/dv/model/otbn_model.cc

```diff
@@ -267,10 +268,10 @@ int OtbnModel::step(svLogic edn_rnd_data_valid,
   assert(!is_xz(edn_urnd_data_valid));
 
   try {
-    if (edn_rnd_data_valid) {
-      uint32_t int_edn_rnd_data[8];
-      set_rnd_data(int_edn_rnd_data, edn_rnd_data);
-      iss->edn_rnd_data(int_edn_rnd_data);
+    // If EDN response interface sends an acknowledgment or RTL signals a valid
+    // ISS forwards the 32b package and the valid signal.
+    if (edn_rnd_data_ack || edn_rnd_data_valid) {
```

Comment:
@rswarbrick

What do think about using `^` instead of ` || ` to guarantee not stepping when `ack` and `valid`  becomes 1 at the same time? We simply ignore if it happens and most likely case would be a cdc assertion fail in `state.py`. I can just check the status at this level but just wondering about your opinion!

Reply:
I'd suggest doing something like what you've got, but then sticking in an explicit assertion that says the two things don't happen at the same time. As you say, they never should, but the assertion helps to document what's going on in the code as well.